.github/workflows/pull-request.yml

```
@@ -0,0 +1,451 @@
+            docs/sidebar*
+            CNAME
+            
+      - name: List all changed unauthorized files
+        if: steps.changed-unauth-files.outputs.any_changed == 'true' || steps.changed-unauth-files.outputs.any_deleted == 'true'
+        env: 
+          CHANGED_UNAUTH_FILES: ${{ steps.changed-unauth-files.outputs.all_changed_files }}
+        run: |
+          for file in ${CHANGED_UNAUTH_FILES}; do
+            echo "$file is unauthorized to change/delete"
+          done
+          echo "To override this, apply the 'ignore-sensitive-files-pr' label"          
```

YAMLlint (1.35.1)

[error] trailing spaces

(trailing-spaces)